### PR TITLE
osmolbp: cli updates & default genesis

### PR DESCRIPTION
**Pull request by harish551**
-  updates to CLI tx commands (create lbp with json input & finalize lbp cmd)
-  added default genesis with params

### CODE REVIEW

File: x/osmolbp/genesis.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+// InitGenesis initializes the capability module's state from a provided genesis
+// state.
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState api.GenesisState) {
+	// k.SetParams(ctx, genState.Params)
+}
+
+// ExportGenesis returns the osmolbp module's exported genesis.
+func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *api.GenesisState {
+	return &api.GenesisState{}
```
[the remark this answers is not in the document]
Comment (robert-zaremba) [reply x/osmolbp/genesis.go:17]: Also, shall we move this function to Keeper (as a method)?

File: x/osmolbp/api/keys.go [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+const (
+	// ModuleName is the module name constant used in many places
+	ModuleName = "osmolbp"
+
+	// RouterKey is the message route for osmolbp
+	RouterKey = ModuleName
+
+	// QuerierRoute is the querier route for osmolbp
+	QuerierRoute = ModuleName
+)
```
Comment (robert-zaremba) [inline x/osmolbp/api/keys.go:12]: Why copying this? we have `x/osmolbp/keys.go`. I think we should not have this consts in `api`.

File: app/app.go [hunk 3]
```diff
@@ -128,6 +128,7 @@ import (
 
 	osmolbpkeeper "github.com/osmosis-labs/osmosis/x/osmolbp/keeper"
 	osmolbpmodule "github.com/osmosis-labs/osmosis/x/osmolbp/module"
+	osmolbptypes "github.com/osmosis-labs/osmosis/x/osmolbp/api"
```
[the remark this answers is not in the document]
Comment (harish551) [reply app/app.go:131]: sure will rename

File: x/osmolbp/genesis.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (harish551) [reply x/osmolbp/genesis.go:17]: will mv genesis file to keepers

File: app/app.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (harish551) [reply app/app.go:131]: we don't need it actually if we remove api/keys.go file

File: x/osmolbp/client/cli/parse.go [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/pflag"
+)
+
+type XCreateLBPInputs createLBPInputs
```
[the remark this answers is not in the document]
Comment (harish551) [reply x/osmolbp/client/cli/parse.go:12]: no reason just copied gamm/pool create cmd parse code and updated
will remove X

File: x/osmolbp/api/keys.go (hunk 2 above)
Comment (robert-zaremba) [reply x/osmolbp/api/keys.go]: yes , that was the idea

File: x/osmolbp/client/cli/parse.go [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/pflag"
+)
+
+type CreateLBPInputs createLBPInputs
```
Comment (robert-zaremba) [inline x/osmolbp/client/cli/parse.go:12]: why type rename?

File: x/osmolbp/client/cli/parse.go [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+type XCreateLBPInputsExceptions struct {
+	CreateLBPInputs
+	Other *string // Other won't raise an error
+}
```
Comment (robert-zaremba) [inline x/osmolbp/client/cli/parse.go:17]: Why do we need this wrapper type? Why do we need `Other *sting`?

If we really need it then it should be a not exported type and we should remove X from the prefix

File: x/osmolbp/client/cli/parse.go (hunk 5 above)
Comment (harish551) [reply x/osmolbp/client/cli/parse.go:12]: without renaming type if i directly use `createLBPInputs`
getting 
`runtime: goroutine stack exceeds 1000000000-byte limit`
I'm not sure why this is happening but with renaming & type conversion it's working

File: x/osmolbp/client/cli/parse.go (hunk 6 above)
Comment (harish551) [reply x/osmolbp/client/cli/parse.go:17]: yeah we can directly use `CreateLBPInputs` type, will remove this

**Review by robert-zaremba — APPROVED** [review]

**Pull request merged**